### ISSUE TITLE
Custom Enchantments

### ISSUE THREAD

**Issue by InspectorGadget** [SpaceDevelopers/CustomEnchantments#1]
You can now comment your suggestions below!

**Comment by DuckIsAwesome** [comment]
Is it like CosmicPe's ComicEnchants

**Comment by InspectorGadget** [comment]
Added few features: https://github.com/SpaceDevelopers/CustomEnchantments/commit/f1cef67cc0e446f7eb51575873e86014eae425fe
According to that, we should add a new system where we can update a certain enchantment. ( only the one which we have ). 
Addition of /ce update [level] to have a system where it checks what of level of enchant you have before updating.

**Comment by JunkDaCoder** [comment]
btw ur gonna have to modify enchantment.php to do this properly

**Comment by InspectorGadget** [comment]
@JunkDaCoder Any suggestion on how to do so? maybe you can commit?

**Comment by JunkDaCoder** [comment]
no, players would have to replace their enchantments.php in order to do so, if they do that, you also have to check the items in anvils so the names dont get replaced
